src/sagemaker/image_uri_config/huggingface.json

```
@@ -913,7 +913,7 @@
                         "us-west-2": "763104351884"
                     },
                     "repository": "huggingface-pytorch-inference",
-                    "container_version": {"gpu": "cu111-ubuntu20.04", "cpu": "ubuntu20.04" }
+                    "container_version": {"gpu": "cu111-ubuntu20.04", "cpu": "ubuntu20.04", "neuron": "sdk1.17.1-ubuntu18.04-v1.0" }
```

```suggestion
                    "container_version": {"gpu": "cu111-ubuntu20.04", "cpu": "ubuntu20.04", "neuron": "sdk1.17.1-ubuntu18.04" }
```
Its probably better to use the image URI without the version in case there might be a security patch or something.
`1.9.1-neuron-py37-sdk1.17.1-ubuntu18.04`

---

+1 for having `huggingface_neuron.json`

---

addressed in current revision.